nutrition_labels/ensemble_grant_tagger.py

```diff
@@ -122,25 +103,32 @@ def output_tagged_grants(self, output_path):
     parser.add_argument(
         '--config_path',
         help='Path to config file',
-        default='configs/ensemble/2021.02.20.ini'
+        default='configs/ensemble/2021.04.01.ini'
     )
 
     args = parser.parse_args()
 
     config = configparser.ConfigParser()
     config.read(args.config_path)
 
-    datestamp = datetime.now().date().strftime('%y%m%d')
+    config_version = "".join(config["DEFAULT"]["version"].split("."))[2:]
 
     grants_data_path = config["prediction_data"]["grants_data_path"]
     input_file_name = os.path.basename(grants_data_path).split('.')[0]
-    output_path = f'data/processed/ensemble/{datestamp}/{input_file_name}_tagged.csv'
+    output_path = f'data/processed/ensemble/{config_version}/{input_file_name}_tagged.csv'
```

Comment:
interesting, do you now version your data like that? this is a good step towards reproducibility